tests/integration/conftest.py

```
@@ -0,0 +1,265 @@
+
+    Identified by container-name prefix (`owui-chat-itest-`), which only ever
+    appears when tests run — no prod chat-id starts with `itest-`. This avoids
+    a production-side env knob just for test cleanup.
+
+    Belt-and-braces: tests usually end their own chat sessions, but a crashed
+    test or a SIGINT mid-suite would otherwise leak containers on the runner.
+    Returns count of containers actually removed.
+    """
+    import sys
+
+    listing = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)